packages/amplication-client/src/Plugins/PluginsCatalog.tsx

```diff
@@ -14,8 +21,28 @@ type Props = AppRouteProps & {
   }>;
 };
 
+type TData = {
+  entities: models.Entity[];
+};
+export const DIALOG_CLASS_NAME = "limitation-dialog";
+const USER_ENTITY_NAME = "user";
+const REQUIRE_AUTH_ENTITY = "requireAuthenticationEntity";
+
 const PluginsCatalog: React.FC<Props> = ({ match }: Props) => {
   const { resource } = match.params;
+  const [confirmInstall, setConfirmInstall] = useState<boolean>(false);
```

Comment:
is it the same one you created before ? maybe it needed in a higher place to shared in those two places